marge/store.py

```diff
@@ -40,12 +40,13 @@ def repo_for_project(self, project):
 
             repo = git.Repo(repo_url, local_repo_dir, ssh_key_file=self._ssh_key_file,
                             timeout=self._timeout, reference=self._reference)
+
             if not self._skip_clone:
                 repo.clone()
-            repo.config_user_info(
-                user_email=self._user.email,
-                user_name=self._user.name,
-            )
+                repo.config_user_info(
+                    user_email=self._user.email,
+                    user_name=self._user.name,
+                )
```

Comment:
This is importanat too. We cannot set the git config if we don't have any repository cloned.